README.md

```diff
@@ -21,17 +21,19 @@ yum install go git tar libpam
 
 ## How to Install in Windows
 ```
+You may install go_ibm_db using either of below commands
 go get -d github.com/ibmdb/go_ibm_db
+go install github.com/ibmdb/go_ibm_db/installer@latest
+go install github.com/ibmdb/go_ibm_db/installer@0.4.1
```

Comment:
Move setup.go to go_ibm_db directory to avoid use of go_ibm_db/installer in installation command.